### ISSUE TITLE
Cannot edit profile properties

### ISSUE THREAD

**Issue by valadas** [dnnsoftware/Dnn.AdminExperience#350]
## Description of bug
Cannot edit profile properties

## Steps to reproduce
List the steps to reproduce the behavior:
1. Go to Site Settings -> Site Behavior -> User Profiles
2. Click the edit icon next to prefix
3. Try to change the length from 50 to 20
4. Click Next

## Current result
The Field name is invalid with a red icon and tooltip saying it is required

## Expected result
Should save.

## Screenshots
![image](https://user-images.githubusercontent.com/6371568/51508059-c1461500-1dc1-11e9-8ab9-ba104b8cbae1.png)

## Error log
None

## Affected version

* [x] 9.3.0 RC0
* [ ] 9.2.2
* [ ] 9.2.1
* [ ] 9.2
* [ ] 9.1.1
* [ ] 9.1
* [ ] 9.0

## Affected browser

Did not try the unchecked ones.
* [x] Chrome
* [ ] Firefox
* [ ] Safari
* [ ] Internet Explorer
* [ ] Edge

**Comment by valadas** [comment]
Got this log:
```
react-dom.development.js?61bb:506 Warning: A component is changing an uncontrolled input of type text to be controlled. Input elements should not switch from uncontrolled to controlled (or vice versa). Decide between using a controlled or uncontrolled input element for the lifetime of the component. More info: https://fb.me/react-controlled-components
    in input (created by SingleLineInput)
    in SingleLineInput (created by SingleLineInputWithError)
    in div (created by SingleLineInputWithError)
    in div (created by SingleLineInputWithError)
    in SingleLineInputWithError (created by ProfilePropertyEditor)
    in div (created by i)
    in i (created by ProfilePropertyEditor)
    in div (created by ProfilePropertyEditor)
    in div (created by GridCell)
    in GridCell (created by GridSystem)
    in div (created by GridSystem)
    in GridSystem (created by ProfilePropertyEditor)
    in div (created by ProfilePropertyEditor)
    in div (created by ProfilePropertyEditor)
    in ProfilePropertyEditor (created by Connect(ProfilePropertyEditor))
    in Connect(ProfilePropertyEditor) (created by ProfilePropertiesPanel)
    in div (created by Motion)
    in div (created by Motion)
    in Motion (created by Collapse)
    in Collapse (created by UnmountClosed)
    in UnmountClosed (created by Collapsible)
    in div (created by Collapsible)
    in Collapsible (created by ProfilePropertyRow)
    in div (created by ProfilePropertyRow)
    in ProfilePropertyRow (created by ProfilePropertiesPanel)
    in div (created by ProfilePropertiesPanel)
    in div (created by ProfilePropertiesPanel)
    in div (created by ProfilePropertiesPanel)
    in ProfilePropertiesPanel (created by Connect(ProfilePropertiesPanel))
    in Connect(ProfilePropertiesPanel) (created by ProfileSettingsPanelBody)
    in div (created by ProfileSettingsPanelBody)
    in ProfileSettingsPanelBody (created by Connect(ProfileSettingsPanelBody))
    in Connect(ProfileSettingsPanelBody) (created by Body)
    in div (created by TabPanel)
    in TabPanel (created by DnnTabs)
    in div (created by UncontrolledTabs)
    in UncontrolledTabs (created by Tabs)
    in Tabs (created by DnnTabs)
    in DnnTabs (created by Body)
    in div (created by TabPanel)
    in TabPanel (created by DnnTabs)
    in div (created by UncontrolledTabs)
    in UncontrolledTabs (created by Tabs)
    in Tabs (created by DnnTabs)
    in DnnTabs (created by Body)
    in div (created by GridCell)
    in GridCell (created by A)
    in div (created by GridCell)
    in GridCell (created by A)
    in A (created by Body)
    in Body (created by Connect(Body))
    in Connect(Body) (created by App)
    in div (created by s)
    in s (created by App)
    in div (created by App)
    in App (created by Connect(App))
    in Connect(App) (created by Root)
    in div (created by Root)
    in Root
    in Provider
```

**Comment by david-poindexter** [comment]
Merged PR to release/1.7.0 - manually closing this related issue
